sql/moz-fx-data-shared-prod/search_terms_derived/search_term_data_validation_reports_v1/init.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS
```

Comment:
@chelseatroy seeing a bunch of errors in this:

s/float/float64/g

and duplicate `num_ranges_compared` with different types

Reply:
Good catch. Fixed!

Where can I see the errors, please? I'd like to be able to fix them in a feedback loop on my own rather than have to ask you if there are any remaining.

Reply:
I just use the [BQ console](https://console.cloud.google.com/bigquery?organizationId=442341870013&project=moz-fx-data-shared-prod&p=moz-fx-data-shared-prod) to run these queries, and it will show errors in queries you are trying to run.